boards/riscv32/hifive1/dts.fixup

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2018 SiFive Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* GPIO 0 */
+#define FE310_GPIO_0_BASE_ADDR	SIFIVE_GPIO0_10012000_BASE_ADDRESS
```

Comment:
We've been using the convention that these should be CONFIG_ prefixed.  So it should be something like:

```
#define CONFIG_FE310_GPIO_0_BASE_ADDR       SIFIVE_GPIO0_10012000_BASE_ADDRESS
```